doc/user/layouts/partials/create-source/connector/s3/details.html

```diff
@@ -1,72 +1,70 @@
+Service][aws-s3], Amazon Web Services' cloud object store. It is important to note that an
 object store behaves differently than a file system does, and that those differences affect how
 Materialize interacts with objects in it.
 
 Some key differences from file systems include:
 
-* Latency on individual S3 operations is much higher than even cloud-network filesystems.
-* It is not possible to seek to a point in an object, the entire object must be downloaded from
-  the beginning in order to read to a point.
-* All object operations are atomic, and it is not possible to modify just part of an S3 object,
+* Latency on individual S3 operations is much higher than on even cloud-network filesystems.
+* It is not possible to seek to a point in an object; Materialize must download the entire object
+  the beginning to read to a point.
+* All object operations are atomic. It is not possible to modify just part of an S3 object;
   you can only entirely replace or delete objects.
 
 The primary effect of this is that we do not handle *updates* to S3 objects, and we may interleave
 multiple object ingestion to speed it up.
 
 [aws-s3]: https://aws.amazon.com/s3/
 
-#### S3 Limitations
+#### S3 limitations
 
 * Currently S3 sources do not support Avro- or Protobuf- encoded objects. Implementation issues:
   [Avro](https://github.com/MaterializeInc/materialize/issues/5596),
   [Protobuf](https://github.com/MaterializeInc/materialize/issues/5597)
 * Object ingest order is not guaranteed.
-* **All S3 sources are append-only** -- deleted and updated [S3 Objects][] are silently ignored.
+* **All S3 sources are append-only**. Deleted and updated [S3 objects][] are silently ignored.
 
 [S3 Objects]: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingObjects.html
 
 #### Object discovery strategies
 
 Materialize has several techniques to discover the objects to download, with [more
-planned][object-specifiers] for the future. Strategies may be specified multiple times within one
-**CREATE SOURCE** statement, so for example this is a legal invocation: `DISCOVER OBJECTS USING BUCKET SCAN
-'example', BUCKET SCAN 'other', SQS NOTIFICATIONS 'example-notifications'`.
+planned][object-specifiers] for the future.
 
-All strategies follow the same basic pattern, `materialized` will:
+All strategies follow the same basic pattern.  Materialize will:
 
-* Interact with a collection of AWS APIs.
-* Deduplicate objects such that the same object is never downloaded twice.
-* Filter the list of objects discovered via those APIs against the [**MATCHING** clause
+* Obtain a list of objects using AWS APIs.
+* Deduplicate objects so the same object is never downloaded twice.
+* Filter the list of objects against the [**MATCHING** clause
   patterns](#patterns).
+* Download the matching objects.
 * Treat each object downloaded as a newline-delimited file for the purposes of record
   delimitation.
```

Comment:
I know I wrote this, but I think this should be "delineation" not "delimitation".